polling_stations/apps/data_finder/views.py

```diff
@@ -27,6 +29,20 @@
 from .helpers.every_election import EmptyEveryElectionWrapper
 
 
+def polling_station_current(station):
+    if not station:
+        return False
+    if not hasattr(station, "elections"):
+        return False
```

Comment:
This means we're only showing data when there's an `IMPORT` type `DataEvent` object  with the `election_dates` field populated.
Is this correct?

Reply:
I don't know right now how much we enforce the election dates, but I think we should do so. In that case, this is ok, I think!